### PR TITLE
doc(newrelic): add doc for new relic account and region from auth config

### PR DESCRIPTION
**Pull request by pragmaticivan**
Signed-off-by: Ivan Santos <301291+pragmaticivan@users.noreply.github.com>

Adds documentation changes to include `account` and `region` via Auth config.
### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #2883

### CODE REVIEW

File: content/docs/2.7/scalers/new-relic.md [hunk 1]
```diff
@@ -68,6 +74,14 @@ spec:
   - parameter: queryKey
     name: new-relic-secret
     key: apiKey
+  secretTargetRef:
```
Comment (JorTurFer) [inline content/docs/2.7/scalers/new-relic.md:77]: `secretTargetRef` is an array of objects with the 3 keys, duplicating the `secretTargetRef` is incorrect. Could you update this?

File: content/docs/2.7/scalers/new-relic.md [hunk 2]
```diff
@@ -68,6 +74,14 @@ spec:
...
+  - parameter: account
+    name: new-relic-secret
+    key: account
+  secretTargetRef:
```
Comment (JorTurFer) [inline content/docs/2.7/scalers/new-relic.md:81]: Same as above

**Review by JorTurFer — CHANGES_REQUESTED** [review]
Thanks for this improvement.
2 comments inline

**Comment by pragmaticivan** [comment]
@JorTurFer Fixed, thanks!

**Review by JorTurFer — APPROVED** [review]
LGTM!
Thanks for this improvemente ❤️

**Review by zroubalik — APPROVED** [review]

**Pull request merged**